lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -340,6 +340,27 @@ void PatchInOutImportExport::processShader() {
       LLPC_OUTS(")\n\n");
     }
   }
+
+  if (m_shaderStage == ShaderStageCompute) {
+    // In a compute shader, process lgc.reconfigure.local.invocation.id calls.
+    // This does not particularly have to be done here; it could be done anywhere after BuilderImpl.
+    for (Function &func : *m_module) {
+      if (func.isDeclaration() && func.getName().startswith(lgcName::ReconfigureLocalInvocationId)) {
+        while (!func.use_empty()) {
+          CallInst *reconfigCall = cast<CallInst>(*func.user_begin());
```

Comment:
Would it be more efficient to use `user_end` here? :)

Reply:
(You mean user_back(), if there is such a thing?) Is it more efficient to remove a use off the end of a def-use chain than off the start?

Reply:
Ah, I didn’t know it was a linked list, then it doesn’t matter.

<sup>I can’t mark my own comments as resolved anymore, so someone else has to do that…</sup>